react_native_webrtc/02_preparations.md

```diff
@@ -0,0 +1,53 @@
+## Preparations
+
+We need to start by preparing ourselves basic React Native application and installing all tools and dependencies we need to start development. You can start a new app using different tools. We've chosen to use [expo](https://expo.dev/) but if you're familiar with React Native you can do whatever leads you to have a 'hello-world' app set up.
+
+### Creating a basic RN application
+
+Let's start by creating the most basic RN application using `create-expo-app`:
+
+```bash
+create-expo-app membrane_rn_webrtc_demo
+```
+
+> We assume that you've got a basic React Native tools installed, but if not here is a good starting point what and how you should prepare to start a development: [https://reactnative.dev/docs/environment-setup](https://reactnative.dev/docs/environment-setup). 
+
+### Installing React Native Membrane WebRTC
+
+Let's go to the lib's GitHub page and see what needs to be done. Since we're using expo we can install it with:
+
+```bash
+expo install @membraneframework/react-native-membrane-webrtc.
+```
+
+Next, add it to `plugins` in `App.json`:
+
+```json
+{
+  "expo": {
+    "name": "membrane_rn_webrtc_demo",
+    ...
+    "plugins": [
+      "@membraneframework/react-native-membrane-webrtc"
+    ],
+    ...
+  }
+}
+```
+
+Now we're ready to get some native code. We can prebuild our application with expo:
+
+```bash
+expo prebuild
```

Comment:
While running that command, I was asked: 
```
What would you like your Android package name to be? ...
```
I couldn't provide a name which would satisfy the naming convention (and apart from the `Wrong package name` error print no information was provided about the naming convention). Finally, I have found out, that the name should consist of two parts, divided wit a dot (something like: `<first_part>.<second_part>`). Perhaps it is obvious for Expo developers, but for me it was a problem - I would suggestion mentioning that here in the tutorial, or linking a comprehensive Expo setup tutorial which would cover that the naming convention on top of this chapter.